.github/workflows/package-dependends-trigger.yml

```diff
@@ -0,0 +1,86 @@
+name: Package Dependends Trigger
+
+on:
+  pull_request_target:
+    types: [labeled, opened, reopened, synchronize]
+
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+  comment-package-dependends-on-issue:
+    name: Comment package dependends on issue
+    runs-on: ubuntu-latest
+    if: "${{ (github.event.action != 'labeled' || github.event.label.name == 'Trigger: Find Dependents') && github.repository_owner == 'e18e' }}"
```

Comment:
@43081j  This `Trigger: FInd Dependents` is the label name & you need to create it. Feel free to suggest other names.